frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.unit.spec.tsx

```diff
@@ -102,4 +115,12 @@ describe("ExpressionEditorSuggestions", () => {
     expect(screen.queryByText("case")).not.toBeInTheDocument();
     expect(screen.queryByText("coalesce")).not.toBeInTheDocument();
   });
+
+  it("should highlight a suggestion when hovering it", () => {
+    const { onHighlightSuggestion } = setup({ startRule: "expression" });
+    expect(screen.getByText("Most used functions")).toBeInTheDocument();
```

Comment:
According to Figma this should say `Common functions`

![image](https://github.com/metabase/metabase/assets/6830683/9cbaab3d-ddee-4aae-962f-b34a159960e8)